<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zchxRadarUi</class>
 <widget class="QWidget" name="zchxRadarUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-size:10pt;
	color:white;
}
QLabel{
	color:white;
	max-height:25px;
	background-color:transparent;
}

QTabWidget{
	background-color: rgb(0,1,127);
}
QTabWidget::pane { /* The tab widget frame */
	border-top: 1px solid #C2C7CB;
	
}

QTabWidget::tab-bar {
      left: 1px; /* move to the right by 5px */
}

  /* Style the tab using the tab sub-control. Note that
      it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
	background: rgb(0,1,127);
      border: 1px solid transparent;
      border-bottom-color: #C2C7CB; /* same as the pane color */
      border-top-left-radius: 2px;
      border-top-right-radius: 2px;
      min-width: 40px;
      padding: 2px;
  }

  QTabBar::tab:selected, QTabBar::tab:hover {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                  stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
  }

  QTabBar::tab:selected {
      border-color: #9B9B9B;
      border-bottom-color: #C2C7CB; /* same as pane color */
	  color:red;
  }

  QTabBar::tab:!selected {
      margin-top: 2px; /* make non-selected tabs look smaller */
  }

  QPushButton{
 max-width:50px;
	 max-height:25px;
	 border:1px solid white;
	 background-color:transparent;
  }

   QPushButton:hover{	 
	 background-color:green;
  }

  QSpinBox,QDoubleSpinBox{
		max-width:50px;
}

#lon, #lat{
	max-width:80px;
}

QLineEdit{
	border:1px solid rgb(180, 180, 180);
	background: transparent;
	border-radius: 2px;
}

QLineEdit::hover{
  color:black;
  border-color:rgb(50,480,40);
  background-color:green;
}
 </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QLabel" name="channel_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>通道一</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tab">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="control_tab">
      <attribute name="title">
       <string>控制</string>
      </attribute>
      <layout class="QVBoxLayout" name="control_tab_layout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="interface_tab">
      <attribute name="title">
       <string>接口</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <property name="spacing">
        <number>1</number>
       </property>
       <item row="0" column="6">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>纬度</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QDoubleSpinBox" name="lat">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="minimum">
          <double>-90.000000000000000</double>
         </property>
         <property name="maximum">
          <double>90.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="5" colspan="2">
        <widget class="QLabel" name="label_36">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>端口</string>
         </property>
        </widget>
       </item>
       <item row="3" column="7">
        <widget class="QSpinBox" name="videoRecPortSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_39">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>状态IP</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="4">
        <widget class="QLineEdit" name="videoRecIPLlineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="4">
        <widget class="QLineEdit" name="controlIPLineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="8" column="7">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="5" colspan="2">
        <widget class="QLabel" name="label_38">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>端口</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_34">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>回波IP</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="4">
        <widget class="QLineEdit" name="heartIPLlineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_31">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>心跳IP</string>
         </property>
        </widget>
       </item>
       <item row="5" column="7">
        <widget class="QSpinBox" name="heartPortSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item row="5" column="5" colspan="2">
        <widget class="QLabel" name="label_37">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>端口</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>中心经度</string>
         </property>
        </widget>
       </item>
       <item row="4" column="7">
        <widget class="QSpinBox" name="controlPortSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>名称</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="7">
        <widget class="QLineEdit" name="name"/>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="lon">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="minimum">
          <double>-180.000000000000000</double>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="7">
        <widget class="QPushButton" name="ok_interface">
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="recongnize_tab">
      <attribute name="title">
       <string>识别</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <property name="spacing">
        <number>1</number>
       </property>
       <item row="0" column="0">
        <widget class="QCheckBox" name="target_length_filter_chk">
         <property name="text">
          <string>目标长度区间(m)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="14" column="4">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>---</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QSpinBox" name="clear_track_time">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>99999999</number>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QSpinBox" name="video_amplitude_max">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>255000</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_29">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>最大轨迹输出</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="target_length_min">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>200000</number>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>最大识别跨度</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>---</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="4">
        <widget class="QSpinBox" name="predictionWidth">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2000000</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QSpinBox" name="target_length_max">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>200000</number>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>---</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="2">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>识别帧数</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QLabel" name="label_15">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>矩形阈值(米)</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>扫描周期(秒)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="4">
        <widget class="QLabel" name="label_27">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>目标清理周期(秒)</string>
         </property>
        </widget>
       </item>
       <item row="13" column="4">
        <widget class="QPushButton" name="ok_parse">
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QSpinBox" name="max_history">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QSpinBox" name="target_area_max">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item row="11" column="4">
        <widget class="QSpinBox" name="target_confirm_spinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QDoubleSpinBox" name="scanCycleTime">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <double>60.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="target_area_filter_chk">
         <property name="text">
          <string>目标面积区间</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="4">
        <widget class="QSpinBox" name="targetMaxSpeedSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="video_amp_chk">
         <property name="text">
          <string>回波阈值</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="video_amplitude_min">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>255000</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="target_area_min">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="move_target_min_speed_label">
         <property name="text">
          <string>最小识别跨度</string>
         </property>
        </widget>
       </item>
       <item row="10" column="4">
        <widget class="QSpinBox" name="move_target_min_speed_spinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_12">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>识别精度</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="video_or_count">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="8" column="4">
        <widget class="QPushButton" name="lap_color_set">
         <property name="text">
          <string>颜色设定</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advance_tab">
      <attribute name="title">
       <string>高级</string>
      </attribute>
      <layout class="QVBoxLayout" name="advance_tab_layout">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
